src/main/java/org/sonar/plugins/stash/StashPluginUtils.java

```diff
@@ -1,15 +1,17 @@
 package org.sonar.plugins.stash;
 
+import com.google.common.base.CharMatcher;
 import java.io.IOException;
+import java.util.Collection;
 import java.util.Properties;
-import org.sonar.api.issue.Issue;
+import org.sonar.api.batch.fs.InputComponent;

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.sonar.api.batch.fs.InputComponent'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)